espnet/asr/pytorch_backend/asr.py

```diff
@@ -694,6 +694,11 @@ def recog(args):
                 new_js[name] = add_results_to_json(js[name], nbest_hyps, train_args.char_list)
 
     else:
+        if hasattr(train_args, "model_module"):
+            if 'transducer' in train_args.model_module:
```

Comment:
This `model_module` is str and it is not type. So this condition is always False. Instead, you can simply put this error in espnet.nets.asr_interface.ASRInterface:
```python
class ASRInterface:
    def recognize_batch(self, ...):
        """Recognize a speech batch."""
        raise NotImplementedError(f"Batch decoding for {type(self).__name__} is not supported yet.")
```

Reply:
Got it. Thanks